src/java/org/apache/cassandra/service/accord/AccordConfigurationService.java

```diff
@@ -350,7 +371,6 @@ void maybeReportMetadata(ClusterMetadata metadata)
     @Override
     public void notifyPostCommit(ClusterMetadata prev, ClusterMetadata next, boolean fromSnapshot)
     {
-        maybeReportMetadata(prev);
```

Comment:
this caused a bug where the first epoch (epoch=5) would add a older epoch (epoch=4) and we wouldn't have anything sync it... so rely on TCM only for `next`